test/test-async.js

```diff
@@ -3722,6 +3722,17 @@ exports['constant'] = function (test) {
     });
 };
 
+exports['constant with multiple arguments'] = function (test) {
```

Comment:
BTW, you don't need to duplicate the test here.  We're migrating away from nodeunit.